http_job.go

```diff
@@ -80,10 +81,12 @@ func (j *httpJob) Error(ctx gocontext.Context, errMessage string) error {
 		return err
 	}
 
-	return j.Finish(FinishStateErrored)
+	return j.Finish(ctx, FinishStateErrored)
 }
 
-func (j *httpJob) Requeue() error {
+func (j *httpJob) Requeue(ctx gocontext.Context) error {
+	context.LoggerFromContext(ctx).WithField("job", j.Payload().Job.ID).Info("requeueing job")
```

Comment:
https://github.com/travis-ci/worker/blob/31dbcbed3e4da64504c087dcff1c05bc8b6eadd9/context/package.go#L53-L55 and https://github.com/travis-ci/worker/blob/31dbcbed3e4da64504c087dcff1c05bc8b6eadd9/processor.go#L141 sets it